tests/components/transmission/test_config_flow.py

```diff
@@ -80,17 +80,21 @@ def init_config_flow(hass):
     return flow
 
 
-async def test_flow_works(hass, api):
+async def test_flow_user_config(hass, api):
     """Test user config."""
-    flow = init_config_flow(hass)
-
-    result = await flow.async_step_user()
+    result = await hass.config_entries.flow.async_init(
+        transmission.DOMAIN, context={"source": "user"}
+    )
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["step_id"] == "user"
 
-    # test with required fields only
-    result = await flow.async_step_user(
-        {CONF_NAME: NAME, CONF_HOST: HOST, CONF_PORT: PORT}
+
+async def test_flow_required_fields(hass, api):
+    """Test with required fields only."""
+    result = await hass.config_entries.flow.async_init(
```

Comment:
Add a pytest fixture that patches `async_setup_entry` in the transmission integration component module. Set the fixture to `autouse=True` to use it for all tests.

Like this:
https://github.com/home-assistant/home-assistant/blob/df04fe32586d446e645819355c40df55e648f9c4/tests/components/hue/test_config_flow.py#L19-L23